src/znc.cpp

```diff
@@ -2121,3 +2152,10 @@ void CZNC::LeakConnectQueueTimer(CConnectQueueTimer* pTimer) {
 }
 
 bool CZNC::WaitForChildLock() { return m_pLockFile && m_pLockFile->ExLock(); }
+
+void CZNC::DisableConfigTimer() {
+  if (m_pConfigTimer) {
+    m_pConfigTimer->Stop();
+    m_pConfigTimer = nullptr;
```

Comment:
No need to recreate the timer every time... `StartMaxCycles(delay, 1)`
Also, manager still has pointer to this timer, which never gets deleted until program shutdown. So number of these times will increase over time.